README.template.md

```diff
@@ -0,0 +1,139 @@
+# [TODO:PROJECT_NAME]
+
+[TODO:PROJECT_DESCRIPTION]
+
+- [Setup](#setup)
+  - [Prerequisites / Dependencies](#prerequisites--dependencies)
+  - [Installation](#installation)
+  - [Post-install](#post-install)
+- [Usage](#usage)
+  - [Running the app](#running-the-app)
+  - [Building the app](#building-the-app)
+- [Testing](#testing)
+  - [Unit testing](#unit-testing)
+  - [End to end tests](#end-to-end-tests)
+  - [Code quality](#code-quality)
+- [Environments](#environments)
+- [Environment variables](#environment-variables)
+  - [List of environment variables](#list-of-environment-variables)
+- [Deployment](#deployment)
+- [Releases](#releases)
+- [Translations](#translations)
+- [Licencing](#licencing)
+
+## Setup
+
+### Prerequisites / Dependencies
+
+You will need to have NodeJS and NPM installed to run this application. It's recommended to use a NodeJS version manager to manage your versions. Required NodeJS versions will typically match LTS (long-term support) versions and you can check which exact version to install by checking `.node-version` file in the repository root.
+
+### Installation
+
+1. clone this repository: `git clone [TODO:GIT_LINK]`
+2. install all the dependencies: `npm ci`
+
+### Post-install
+
+1. set up the [environment variables](#environment-variables)
+
+## Usage
+
+### Running the app
+
+To run the app, you can simply run `npm start`. This starts the application in the environment dependent on your environment setup.
```

Comment:
With next.js projects, this will generally be `npm run dev`

Reply:
Start is a generic wording, I would keep that as is, and adapt it to the project stack. This file should be kept stack-agnostic.